thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/EmailResource.java

```diff
@@ -27,55 +25,24 @@
 import javax.ws.rs.core.Response;
 
 import com.linkedin.thirdeye.common.ThirdEyeConfiguration;
-import com.linkedin.thirdeye.datalayer.bao.AnomalyFunctionManager;
-import com.linkedin.thirdeye.datalayer.bao.EmailConfigurationManager;
-import com.linkedin.thirdeye.datalayer.dto.AnomalyFunctionDTO;
-import com.linkedin.thirdeye.datalayer.dto.EmailConfigurationDTO;
 import com.linkedin.thirdeye.datasource.DAORegistry;
 
-import org.apache.commons.lang3.StringUtils;
-
 @Path("thirdeye/email")
 @Produces(MediaType.APPLICATION_JSON)
 public class EmailResource {
 
-  private final AnomalyFunctionManager functionDAO;
-  private final EmailConfigurationManager emailDAO;
   private final AlertConfigManager alertDAO;
   private static final DAORegistry DAO_REGISTRY = DAORegistry.getInstance();
   private ThirdEyeConfiguration thirdeyeConfiguration = null;
   private AlertFilterFactory alertFilterFactory;
 
   public EmailResource(ThirdEyeConfiguration thirdeyeConfiguration) {
-    this.functionDAO = DAO_REGISTRY.getAnomalyFunctionDAO();
-    this.emailDAO = DAO_REGISTRY.getEmailConfigurationDAO();
     this.alertDAO = DAO_REGISTRY.getAlertConfigDAO();
     this.thirdeyeConfiguration = thirdeyeConfiguration;
     this.alertFilterFactory = new AlertFilterFactory(this.thirdeyeConfiguration.getAlertFilterConfigPath());
   }
 
   @POST
-  public Response createEmailConfig(EmailConfigurationDTO emailConfiguration) {
```

Comment:
Its better that you explain more in your pull request description what you are cleaned up here and make sure no one are actually using these endpoints.